scalafmt-tests/src/test/resources/unit/Comment.stat

```diff
@@ -201,3 +201,150 @@ object literal extends scala.Dynamic { // scalastyle:ignore
 object literal extends scala.Dynamic { // scalastyle:ignore
   ???
 }
+<<< #1234 1: single-line
+comments.wrap = standalone
+===
+// A really long comment that scalafmt should break up but does not because this feature is not implemented yet
+// short comment
+object a {
+  // short comment
+  // Another long comment demonstrating that wrapped comments will be aligned sensibly
+  def a = {}
+}
+>>>
+/* A really long comment that scalafmt
+ * should break up but does not because
+ * this feature is not implemented yet */
+// short comment
+object a {
+  // short comment
+  /* Another long comment demonstrating
```

Comment:
Personally I prefer it to be formatted as 

```
// Another long comment demonstrating
// that wrapped comments will be
// aligned sensibly
```

but this wrap is very useful too

Reply:
done. i didn't think this would be the case, as to me each `//` line usually represents a separate comment.